docker/nodes/postgres/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM jenkins/inbound-agent:alpine
+USER root
+WORKDIR /opt
+# RUN apk update && apk add curl postgresql gnupg && \
```

Comment:
Should this commented-out code be reinstated?

Reply:
Good spot, I think I was trying to make the build faster when testing.